### PR TITLE
fix height of WebsiteTabs links

### PR DESCRIPTION
**Pull request by lovemecomputer**
closes https://github.com/carbon-design-system/carbon-website/issues/1426

### CODE REVIEW

**Comment by designertyler** [comment]
@lovemecomputer I'm still seeing some wonky issues in the deploy preview. The tabs are 1px too low and the height it 44px. The tabs should be 40px tall. 
Screen Shot 2019-04-15 at 2 12 16 PM

**Comment by shixiedesign** [comment]
Yeah pretty much what Tyler said. Just to illustrate, the 40px is including the 3px blue bar on top. Maybe that's how the 43px come to be. 
image

**Comment by lovemecomputer** [comment]
@tay1orjones thanks! gonna verify a detail with you on slack

**Comment by lovemecomputer** [comment]
@shixiedesign thanks!!

File: src/components/WebsiteTabs/website-tabs.scss [hunk 1]
```diff
@@ -7,39 +7,45 @@
   margin-bottom: 0;
 }
 
-.website-tabs nav.bx--tabs {
+.website-tabs nav.#{$prefix}--tabs {
   min-height: 2.8rem;
```
Comment (alisonjoseph) [inline src/components/WebsiteTabs/website-tabs.scss]:
```suggestion
  min-height: 2.5rem;
```

File: src/components/WebsiteTabs/website-tabs.scss [hunk 2]
```diff
@@ -7,39 +7,45 @@
...
-.website-tabs a.bx--tabs__nav-link {
+.website-tabs a.#{$prefix}--tabs__nav-link {
+  display: block;
+  height: 100%;
   border: none;
   box-sizing: border-box;
   background: $ui-03;
   color: $text-01;
   line-height: 1.5;
   padding: $spacing-sm $spacing-md;
```
Comment (alisonjoseph) [inline src/components/WebsiteTabs/website-tabs.scss]:
```suggestion
  padding: $spacing-sm $spacing-md;
  height: 2.5rem;
```

I think we need to set the height here too

File: src/components/WebsiteTabs/website-tabs.scss (hunk 1 above)
Comment (lovemecomputer) [reply src/components/WebsiteTabs/website-tabs.scss]: thanks!

File: src/components/WebsiteTabs/website-tabs.scss (hunk 2 above)
Comment (lovemecomputer) [reply src/components/WebsiteTabs/website-tabs.scss]: puttin this in too, thanks!

**Comment by lovemecomputer** [comment]
@alisonjoseph you're right, i missed that! tweaked the spacing on the selected item for the fixed height and all should be good : )

**Pull request merged**